packages/blocks/src/page-block/default/default-page-block.ts

```diff
@@ -248,7 +248,7 @@ export class DefaultPageBlockComponent
       return;
     } else if (e.key === 'ArrowDown' && hasContent) {
       e.preventDefault();
-      const firstParagraph = model.children[0].children[0];
+      const firstParagraph = model.children[1].children[0];
```

Comment:
/cc @Saul-Mirone do you have a recommended way to query the first paragraph here?

Reply:
Please check the flavour of block.